ggircs-app/app/components/XmlDiff/DiffDetails.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Row, Col } from "react-bootstrap";
+
+interface Props {
+  report: any;
+}
+
+export const DiffDetails: React.FunctionComponent<Props> = ({ report }) => (
```

Comment:
Maybe we could have three `string` props instead, to avoid a complex object with `any` type